commerce-taglib/src/main/java/com/liferay/commerce/taglib/servlet/taglib/PriceTag.java

```diff
@@ -353,30 +361,29 @@ protected void setProductInfo() throws Exception {
 			}
 		}
 
-		if (_cpInstanceId > 0) {
-			_cpInstance = CPInstanceServiceUtil.getCPInstance(_cpInstanceId);
-		}
-		else {
-			if (cpDefinition.isIgnoreSKUCombinations()) {
-				CPInstanceHelper cpInstanceHelper =
-					ServletContextUtil.getCPInstanceHelper();
+		if (!(_cpInstanceId > 0) && cpCatalogEntry.isIgnoreSKUCombinations()) {
```

Comment:
`!(_cpInstanceId > 0)` should be simplified to `(_cpInstanceId <= 0)` or `(_cpInstanceId == 0)` (whichever you like better)

Reply:
See https://github.com/liferay/com-liferay-commerce/pull/1414